dags/aps/APSParams.py

```diff
@@ -0,0 +1,23 @@
+class APSParams:
+    def __init__(
+        self,
+        from_date: str = '2000-01-01',
+        until_date: str = '2000-01-02',
+        date: str = 'published',
+        journals: str = '',
+        sets: str = '',
+        per_page: int = 100
+    ):
+        self.from_date = from_date
+        self.until_date = until_date
+        self.date = date
+        self.journals = journals
+        self.sets = sets
+        self.per_page = per_page
+
+    def get_params(self) -> dict:
+        params = {'from': self.from_date, 'until': self.until_date, 'date': self.date, 'journals': self.journals, 'sets': self.sets, 'per_page': self.per_page}
+        print(params)
```

Comment:
is tis for debugging?

Reply:
yes

Reply:
you could use logging for that with `debug` level